Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Core.UnitTests/Common/Assertions/ErrorOr/ErrorOrAssertions.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace EvolutionaryArchitecture.Fitnet.Contracts.Core.UnitTests.Common.Assertions.ErrorOr;
+
+using FluentAssertions;
```

Comment:
You do not need it here, it is defined in global

Reply:
Can we move it to Common package?

Reply:
moved to global.

Reply:
We will move in next PR to common